internal/jobs/ctx.go

```diff
@@ -18,3 +21,25 @@ func copyContext(ctx context.Context) context.Context {
 	nCtx = identity.WithAccountId(nCtx, identity.AccountId(ctx))
 	return nCtx
 }
+
+func jobContextLogger(origCtx context.Context, principal identity.Principal, traceID, edgeID string, jobID uuid.UUID, accountID, reservationID int64) (context.Context, *zerolog.Logger) {
+	ctx := logging.WithJobId(origCtx, jobID.String())
+	ctx = logging.WithReservationId(ctx, reservationID)
+	ctx = identity.WithIdentity(ctx, principal)
+	ctx = logging.WithTraceId(ctx, traceID)
+	ctx = logging.WithEdgeRequestId(ctx, edgeID)
+	ctx = identity.WithAccountId(ctx, accountID)
+
+	logger := zerolog.Ctx(ctx)
+	logger = ptr.To(logger.With().
+		Int64("account_id", accountID).
+		Str("org_id", principal.Identity.OrgID).
+		Str("account_number", principal.Identity.AccountNumber).
+		Str("trace_id", traceID).
+		Str("request_id", edgeID).
+		Int64("reservation_id", reservationID).
+		Str("job_id", jobID.String()).
+		Logger())
+	ctx = logger.WithContext(ctx)
+	return ctx, logger
+}
```

Comment:
Just saw your PR and realized some of this code should live in worker.contextLogger instead. Moving.